src/main/avs-util/error.c

```diff
@@ -96,4 +96,14 @@ const char *avs_util_error_str(avs_error error)
     }
 
     return avs_util_error_unknown;
+}
+
+const char *avs_util_property_error_get_and_clear(struct property *prop)
+{
+    avs_error error;
+
+    error = property_get_error(prop);
+    property_clear_error(prop);
```

Comment:
this returns a property* per the earlier stacked commit, is this something we need to be considering?

Reply:
Nope, I fixed this in https://github.com/djhackersdev/bemanitools/pull/299